#! /bin/sh /usr/share/dpatch/dpatch-run
## 29_scripts__mysqlbug.sh.dpatch by  <ch@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: scripts__mysqlbug.sh

@DPATCH@

#
# 1. Debian specific defaults
#
--- old/scripts/mysqlbug.sh.orig	2004-05-31 23:31:42.000000000 +0200
+++ new/scripts/mysqlbug.sh	2004-05-31 23:32:07.000000000 +0200
@@ -132,7 +132,9 @@
 then
   if test -z "$EDITOR"
   then
-    EDIT=emacs
+    # EDIT=emacs
+    # in Debian, we have the sensible-editor symlink. -ch-
+    EDIT=sensible-editor
   else
     EDIT="$EDITOR"
   fi
